code/modules/admin/topic.dm

```diff
@@ -11,9 +11,29 @@
 		return TRUE
 	log_admin_private("[key_name(usr)] clicked an href with [msg] authorization key! [href]")
 
-/datum/admins/Topic(href, href_list)
-	..()
+// BASE ADMINS_TOPIC
+
+/datum/admins_topic()
+	var/keyword
+	var/log = TRUE

```

Comment:
If I remember correctly this should probably be replaced by something like LOG_ADMIN_PRIVATE and LOG_ADMIN_PUBLIC